core/src/test/scala/unit/kafka/server/DynamicConfigChangeTest.scala

```diff
@@ -327,22 +253,6 @@ class DynamicConfigChangeTest extends KafkaServerTestHarness {
       } finally {
         admin.close()
       }
-    } else {
```

Comment:
Could you please also remove `if (isKRaftTest())` for this test? It is redundant now.